### DISCUSSION TITLE
Javascript autocomplete

### DISCUSSION THREAD

**Discussion by LukineskoSK** [microsoft/vscode-discussions#685] [Extension Development QnA] [answered]
Hello why when I type "document.querySelector(".diceImage")" and then I go to put .style it doesn't auto prompt me ? Is there any extension ?

**Comment by imbant** [comment, marked as answer]
Try `document.querySelector<HTMLElement>(".diceImage")` if you use typescript.
`style` is the property of the [HTMLElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style). But querySelector returns [Element](https://developer.mozilla.org/en-US/docs/Web/API/Element) by default

**Comment by isidorn** [comment]
Closing as this community is for extension authors.
